corehq/motech/repeaters/tests/test_models.py

```diff
@@ -51,7 +52,7 @@ def setUp(self):
         return super().setUp()
 
     def tearDown(self):
-        delete_all_repeaters()
+        # delete_all_repeaters()
```

Comment:
?

Reply:
Thanks for noticing that. It was probably missed while testing something.